core/src/main/resources/hudson/slaves/JNLPLauncher/main.jelly

```diff
@@ -104,8 +76,28 @@ ${copy_secret_to_file}
 ${copy_agent_jar_windows}
 ${copy_java_cmd_secret2_windows}
 </pre>
-            </j:otherwise>
-          </j:choose>
+          <j:if test="${!it.launcher.configured}">
+              <p>
+                  ${%commonOptions}
+              </p>
+          </j:if>
+          <j:if test="${!it.launcher.descriptor.tcpSupported}">
+              <p>
+                  ${%tcp-port-disabled}
+                  <l:isAdmin><a href="${rootURL}/manage/configureSecurity/">${%configure.link.text}</a>.</l:isAdmin>
+              </p>
```

Comment:
Similar to older message, but not marked as an error since it may be perfectly normal in a WebSockets-only installation.